uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/test/shared/components/leave-calendar-day.component.spec.js

```diff
@@ -218,86 +225,13 @@ define([
 
           absenceType.calculation_unit = calculationUnitInHours;
           controller.date = startDate.add(1, 'days').format(dateFormat);
-          leaveRequest.to_date = startDate.add(2, 'days').format(dateFormat);
-          $rootScope.$digest();
-        });
-
-        it('sets day label equal to empty string', function () {
-          expect(controller.label).toBe('');
-        });
-      });
-    });
-
-    describe('selecting a tooltip template', function () {
```

Comment:
I'm guessing the tooltip selection logic is going to be added in the next PR, right?

Reply:
No, the tooltip selection logic has been moved to UI, it is really simple and depends on leaveRequest only. I should have mentioned that in comments to the PR, let me do that.